src/components/accordionCo.js

```diff
@@ -0,0 +1,135 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { makeStyles } from "@material-ui/core/styles"
+import clsx from "clsx"
+import Collapse from "@material-ui/core/Collapse"
+import IconButton from "@material-ui/core/IconButton"
+import Typography from "@material-ui/core/Typography"
+import ArrowDropDownIcon from "@material-ui/icons/ArrowDropDown"
+import palette from "../theme/palette"
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: "100%",
+  },
+  summary: {
+    display: "flex",
+    flexDirection: "row",
+    padding: "33px 0 31px 0",
```

Comment:
maybe is prettier to use such as:

````
padding: [33, 0, 31, 0],
````

Reply:
Hola! Algo que me gustaría comentar es que los valores de espaciado siempre deben ser múltiplos de 8 (En este caso el espaciado debería ser 32px arriba y 32px abajo). Más allá, una sugerencia que tengo es utilizar los [helpers de espaciado de Material-UI](https://material-ui.com/customization/spacing/) para configurar espacios. De esa forma, se configuran valores siempre en función de múltiplos de 8.

Reply:
It can`t resolve in this form